### PR TITLE
Upgrade NuGet packages

### PR DESCRIPTION
**Pull request by IEvangelist**

### CODE REVIEW

**Comment by Youssef1313** [comment]
CI failure is addressed by https://github.com/dotnet/docs-tools/pull/51

File: actions/docs-verifier/src/MarkdownLinksVerifier/MarkdownFilesAnalyzer.cs [hunk 1]
```diff
@@ -37,13 +38,17 @@ public static class MarkdownFilesAnalyzer
                 }
 
                 MarkdownDocument document = Markdown.Parse(await File.ReadAllTextAsync(file), s_pipeline);
-                foreach (LinkInline link in document.Descendants<LinkInline>())
+                foreach (LinkInline link in
+                    document.Descendants<LinkInline>().Where(link => link.Url is not null))
```
Comment (Youssef1313) [inline actions/docs-verifier/src/MarkdownLinksVerifier/MarkdownFilesAnalyzer.cs:42]: Do we have cases where `link.Url` is null?

**Review by Youssef1313 — APPROVED** [review]
LGTM, with a small question.

File: actions/docs-verifier/src/MarkdownLinksVerifier/MarkdownFilesAnalyzer.cs (hunk 1 above)
Comment (IEvangelist) [reply actions/docs-verifier/src/MarkdownLinksVerifier/MarkdownFilesAnalyzer.cs:42]: No, this is not related to an issue but rather a compiler warning (on a clean build).

**Review by BillWagner — APPROVED** [review]
This LGTM @IEvangelist 

You can :shipit: when ready.

**Pull request merged**